suites/pacific/rbd/tier-1_rbd_mirror.yaml

```diff
@@ -28,6 +28,8 @@ tests:
       desc: Setup phase to deploy the required pre-requisites for running the tests.
       module: install_prereq.py
       abort-on-fail: true
+      config:
```

Comment:
please remove this config

Reply:
will remove, added as part of testing . just now latest run is completed 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-phzsy/